### PR TITLE
docs: add js release notes

### PR DESCRIPTION
**Pull request by aslushnikov**

### CODE REVIEW

**Review by dgozman — APPROVED** [review]

File: docs/src/release-notes-js.md [hunk 1]
```diff
@@ -5,6 +5,132 @@ title: "Release notes"
...
+    // ...
+  });
+
+  test('two', async ({ page }) => {
+    // ...
+  });
+});
+```
+
+### 🌍 Multiple Web Servers in `playwright.config.ts`
```
Comment (rwoll) [inline docs/src/release-notes-js.md:104]: Consider making this the first item in the notes. We get questions about this frequently in Slack, and the issue has decent traction.

File: docs/src/release-notes-js.md [hunk 2]
```diff
@@ -5,6 +5,132 @@ title: "Release notes"
...
+
+Launch multiple web servers by passing an array of configurations:
```
Comment (rwoll) [inline docs/src/release-notes-js.md:106]:
```suggestion
Launch multiple web servers, databases, or other processes by passing an array of configurations:
```

**Review by rwoll — APPROVED** [review]
LGTM. Left a few suggestions!

File: docs/src/release-notes-js.md (hunk 1 above)
Comment (aslushnikov) [reply docs/src/release-notes-js.md]: Good idea!

**Pull request merged**